force-app/main/default/classes/AFFL_MultiRecordType_TDTM.cls

```diff
@@ -212,31 +318,59 @@ public class AFFL_MultiRecordType_TDTM extends TDTM_Runnable {
             for (String primaryField : afflMapper.primaryAfflFieldNames) {
                 dynamicQuery = dynamicQuery + ', ' + primaryField;
             }
-            dynamicQuery += ' from Contact where ID IN '+ contactIDs;
+            dynamicQuery += ' from Contact where ID IN ' + contactIDs;
             UTIL_Debug.debug('****MRT: Dynamic query: ' + dynamicQuery);
             if (contactIDs != '()') {
                 List<Contact> relatedContactsList = Database.query(dynamicQuery);
-                for(Contact contact : relatedContactsList) {
+                for (Contact contact : relatedContactsList) {
                     relatedContactsMap.put(contact.ID, contact);
                 }
             }
         }
         return relatedContactsMap;
     }
 
-    private void populateKeyAffls(Affiliation__c affl, String lookupFieldAPIName, Contact relatedContact, DmlWrapper dmlWrapper) {
+    /*******************************************************************************************************
+     * @description Populates the primary affiliation field for a Contact from an Affiliation's Account
+     * @param affl The Primary Affiliation being modified.
+     * @param lookupFieldAPIName The api name of the Primary Affiliation lookup field
+     * @param relatedContact The Contact to populate the Primary Affiliation field for
+     * @param dmlWrapper The DmlWrapper to register the change to
+     * @return The modified contact
+     *******************************************************************************************************/
+    @TestVisible
+    private Contact populateKeyAffls(
+        Affiliation__c affl,
+        String lookupFieldAPIName,
+        Contact relatedContact,
+        DmlWrapper dmlWrapper
+    ) {
         //If the reason why Affiliations have been made primary is that key affiliation fields on Contact have been populated,
         //we don't need to try populating them. In fact, this causes an error notification to be sent (W-009272).
-        if (!TDTM_ProcessControl.getRecursionFlag(TDTM_ProcessControl.registeredTrigger.CON_PrimaryAffls_TDTM_keyAfflLookupUpdated)) {
+        if (
+            !TDTM_ProcessControl.getRecursionFlag(
+                TDTM_ProcessControl.registeredTrigger.CON_PrimaryAffls_TDTM_keyAfflLookupUpdated
+            )
+        ) {
             //If we have a mapping, populate the lookup field defined in the mapping.
             if (!String.isBlank(lookupFieldAPIName)) {
                 relatedContact.put(lookupFieldAPIName, affl.Account__c);
                 dmlWrapper.objectsToUpdate.add(relatedContact);
+                return relatedContact;
             }
         }
+
+        return null;
     }
 
-    public void uncheckOtherPrimariesSameType(List<Affiliation__c> affls, DmlWrapper dmlWrapper) {
+    /*******************************************************************************************************
```

Comment:
Changes on this method include adding apex doc comments and having the method return a value for testability.